architecture/adr-002-sub-zones.md 

```diff
@@ -0,0 +1,179 @@
+    (gogoproto.nullable) = false
+  ];
+  int64 unbonding_period = 22;
+  int64 messages_per_tx = 23;
+  int64 decimals = 24;
+  bool unbonding_enabled = 25;
+  bool deposits_enabled = 26;
+  bool return_to_sender = 27;
+  bool is_118 = 28;
+}
+```
+
+Alternatives
+
+Sub-zones could also be implemented as an array of "subzoneID-authority" tuples that are stored themselves on the parent zone.
+This would mean that the existing member functions of a `Zone` could not be directly accessed by a sub zone, making things a bit more complicated in the implementation.
+This alternative would be more clearer that a sub-zone is actually not a new "zone" meaning chain.
+
+Decision
+
+We will add the following fields to the `Zone` struct:
+
+```proto
+message Zone {
+  string connection_id = 1;
+  string chain_id = 2;
+  ICAAccount deposit_address = 3;
+  ICAAccount withdrawal_address = 4;
+  ICAAccount performance_address = 5;
+  ICAAccount delegation_address = 6;
+  string account_prefix = 7;
+  string local_denom = 8;
+  string base_denom = 9;
+  string redemption_rate = 10 [
+    (cosmos_proto.scalar) = "cosmos.Dec",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable) = false
+  ];
+  string last_redemption_rate = 11 [
+    (cosmos_proto.scalar) = "cosmos.Dec",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable) = false
+  ];
+  repeated Validator validators = 12;
+  repeated ValidatorIntent aggregate_intent = 13 [(gogoproto.castrepeated) = "ValidatorIntents"];
+  bool multi_send = 14; // deprecated
+  bool liquidity_module = 15;
+  uint32 withdrawal_waitgroup = 16;
+  bytes ibc_next_validators_hash = 17;
+  uint64 validator_selection_allocation = 18;
+  uint64 holdings_allocation = 19;
+  int64 last_epoch_height = 20; // deprecated
+  string tvl = 21 [
+    (cosmos_proto.scalar) = "cosmos.Dec",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable) = false
+  ];
+  int64 unbonding_period = 22;
+  int64 messages_per_tx = 23;
+  int64 decimals = 24;
+  bool unbonding_enabled = 25;
+  bool deposits_enabled = 26;
+  bool return_to_sender = 27;
+  bool is_118 = 28;
+  SubzoneInfo subzoneInfo = 29;
+}
+
+message SubzoneInfo {
+      string authority = 1;
+      string base_chainID = 2;
+}
+```
+The `SubzoneInfo` `authority` field is the whilelisted Quicksilver account which controls this subzone.  The `BaseChainID`
+field is a reference to the "base" or "parent" chain that this zone is a sub-zone of.  For example, if a base zone for
+the Cosmos Hub with chainID "gaia-5" exists, an `authority` could propose to create a new `sub-zone` with "gaia-5" as its
+base zone.
+
+If `SubzoneInfo` and is non-empty, then the zone is a subzone.  We can add the following helper function:
+
+```go
+func (z *Zone) IsSubzone() bool {
+    return z.SubzoneInfo.Authority != "" && z.SubzoneInfo.BaseChainID != ""
```

Comment:
Is this required; or can we just do `return z.SubZoneInfo != nil`

The create subzone proposal type will ensure that any subzoneInfo must be created with both a baseChainID and an Authority.